app/main.py

```diff
@@ -16,11 +17,13 @@ class Config:
     _DEFAULTS = {
         'DOWNLOAD_DIR': '.',
         'AUDIO_DOWNLOAD_DIR': '%%DOWNLOAD_DIR',
+        'CUSTOM_DIRS': 'true',
+        'CREATE_DIRS': 'false',

```

Comment:
I would remove CUSTOM_DIRS altogether (I don't see a use case where it wouldn't be true), and would set CREATE_DIRS by default to true. (And perhaps change the name to be more descriptive, since CREATE_DIRS isn't telling enough, maybe ALLOW_DOWNLOAD_DIR_CREATE? Or something similar if you can think of something shorter :) )